configs/textdet/_base_/datasets/sroie.py

```diff
@@ -0,0 +1,15 @@
+sroie_textdet_data_root = 'data/sroie'
```

Comment:
This file can be deleted. `sroie` is not a prevailing dataset in OCR, and therefore we don't need to provide these base configs by default. Config generator will generate these base configs when needed.